ide-support/revliburl.livecodescript

```diff
@@ -3333,7 +3333,7 @@ on libUrlFtpUpload pData,pUrl,pMessage
   
   put false into lvJumpOut
   put ulStripUrl(pUrl) into newUrl
-  if char 1 to 3 of newUrl <> "ftp" then
+  if char 1 to 3 of newUrl <> "ftp" and char 1 to 4 of newUrl <> "sftp" then
```

Comment:
@techstrategies it seems we could do better with the protocol check here by splitting on : and then checking the protocol matches completely rather than just the first x chars.